### PR TITLE
Poison Ivy: Slow Falling and Rename

### PR DESCRIPTION
**Pull request by weluvgoatz**
In this pull request, poison ivies have been renamed to "Spring Leaf", since they aren't really poison ivies. 

Also, poison ivies now float and fall slowly, just like a leaf in real life.

### CODE REVIEW

**Comment by Rusty-Box** [comment]
The idea is fine but I think it should get proper floating sprites before this gets merged because otherwise it is a bit odd looking.

File: src/badguy/poisonivy.cpp [hunk 1]
```diff
@@ -26,14 +26,29 @@
 
 PoisonIvy::PoisonIvy(const ReaderMapping& reader)
   : WalkingBadguy(reader, "images/creatures/poison_ivy/poison_ivy.sprite", "left", "right")
-{
+ {
   walk_speed = 80;
-}
+ }
```
Comment (HybridDog) [inline src/badguy/poisonivy.cpp]: Please don't add spaces before the braces.

File: src/badguy/poisonivy.hpp [hunk 2]
```diff
@@ -18,21 +18,25 @@
...
   virtual bool is_freezable() const override;
   virtual std::string get_class() const override { return "poisonivy"; }
-  virtual std::string get_display_name() const override { return _("Poisonous ivy"); }
+  virtual std::string get_display_name() const override { return _("Spring Leaf"); }
 
 protected:
   virtual bool collision_squished(GameObject& object) override;
 
 private:
+  Physic physic;
```
Comment (HybridDog) [inline src/badguy/poisonivy.hpp]: It looks like `physic` is unused. I think that `m_physic` is from some super class.

**Comment by weluvgoatz** [comment]
@Rusty-Box I've added support for a falling animation.  You can always change it when you make the new poision ivy sprite, this sprite is just a placeholder.

**Comment by weluvgoatz** [comment]
Closing this, it's not ready yet.

**Comment by Rusty-Box** [comment]
Reopening, as it is still considered but has to wait for required graphics. Closing might have us forgot that this PR still gets merged in the future.

**Comment by Alzter** [comment]
Idea: When Poison Ivy falls it turns into a flat leaf, and sways and tilts left to right as it glides to the ground.

**Comment by busybody42** [comment]
no... This is a unique idea, but poison ivies are a better enemy... also, spring leaf could be a bouncing snowball for the forest, but otherwise.... yeah, no... but the slow falling part IS ok, but could break levels...

**Comment by weluvgoatz** [comment]
Closing this seen as it has been open 8 months, and I'm working on different things rather than this, which I would like to remain out of my mind.  Since we have a design document now, I added this idea to that so when the time comes we can revisit this, but as for now, I'm closing it.

**Pull request closed without merging**